README.md

```diff
@@ -1,2 +1,97 @@
+| --- | --- |
+| The Adobe Experience Platform **Launch runtime** is designed to support the following browsers:| **The tag runtime** in Adobe Experience Platform is compatible with multiple browsers: |
+| When users install an extension to an Adobe Experience Platform **Launch property**, they will be shown the extension configuration view which your extension will provide.| When users install an extension to **a tag property**, they will be shown the extension configuration view which your extension will provide.|
+
+When tags are providing the functionality to the non-specific object instance of the sentence (as opposed to Platform), the qualifier will be plural as it is referring to the suite of technologies encompassed by the term "tags". In this case, it makes more grammatical sense to add a hyphenated verb to improve the clarity of the sentence.
+
+In the example sentences, the hyphenated "enabled" was added to differentiate between the functionality provided by tags to the nouns, and that provided by Platform.
+
+| Original Example | Updated sentence |
+| --- | --- |
+| This extension provides the functionality for adding the `Media` tracker instance to **an Adobe Experience Platform Launch site** or project.  | This extension provides the functionality for adding the `Media` tracker instance to **a tags-enabled site** or project.   |
+| After you have included all three of the extensions mentioned above in your **Platform Launch project**, you can proceed in one of two ways:  | After you have included all three of the extensions mentioned above in your **tags-enabled project**, you can proceed in one of two ways:  |
+
+### Referring to extensions (tag extensions or event-forwarding extensions)
+
+Writing about extensions for both tags and event forwarding capabilities follows the same format as dealing with common nouns. The name of the extension (proper noun) will precede the qualifier (either tag or event forwarding) followed by the word "extension" (common noun). 
+
+The branding changes regarding event forwarding are handled in the same way as tags because they too are a suite of technologies and not a branded service. Another example is provided below. This format can be seen in the following examples. 
+
+<!-- Get a better example for event forwarding tags seen below -->
+
+| Original Example | Updated sentence |
+| --- | --- |
+| Learn about **the Adobe ContextHub extension in Adobe Experience Platform Launch**. | Learn about **the Adobe ContextHub tag extension in Adobe Experience Platform**. |
+| Learn about **the Server Side Cloud Connector extension** in Adobe Experience Platform Launch.| Learn about **the Cloud Connector event forwarding extension** in Adobe Experience Platform. |
+
+The context provided by "tag extension" or "event forwarding" negates the need for the wording "in Adobe Experience Platform", however it can be used to provide further contextual clarity on the first instance within the document.
```

Comment:
Not sure why we need to drive this point home in particular. Why don't we just remove "in Adobe Experience Platform" from the examples so we don't have to go out of our way to explain this? We can just remove this line after.